.github/workflows/ci-pipeline.yml

```diff
@@ -17,4 +23,47 @@ jobs:
     - name: "Unit Tests"
       run: docker-compose -f docker-compose-test-unit.yml up --exit-code-from todo-app
     - name: "End to end Tests"
-      run: docker-compose -f docker-compose-test-e2e.yml up --exit-code-from todo-app
\ No newline at end of file
+      run: docker-compose -f docker-compose-test-e2e.yml up --exit-code-from todo-app
+
+  release:
+    name: Release to Production
+    runs-on: ubuntu-latest
+    env:
+      HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
+    if: ${{ github.ref_name == 'main' }}
+    needs: test
+    steps: 
+    - name: "Checkout code"
+      uses: actions/checkout@v2
+
+    - name: "Create env file"
+      run: |
+        touch .env
+        echo "${{ secrets.SERVER_ENV }}" > .env
```

Comment:
What's this for? It doesn't look like you're putting the `.env` file in your Docker image or anything (that would be bad since it's public). Does it work if you remove it?